src/odemis/acq/test/acq_test.py

```diff
@@ -486,5 +484,302 @@ def on_progress_update(self, future, start, end):
         self.end = end
         self.updates += 1
 
+class CRYOSECOMTestCase(unittest.TestCase):
+    backend_was_running = False
+
+    @classmethod
+    def setUpClass(cls):
+
+        try:
+            test.start_backend(ENZEL_CONFIG)
+        except LookupError:
+            logging.info("A running backend is already found, skipping tests")
+            cls.backend_was_running = True
+            return
+        except IOError as exp:
+            logging.error(str(exp))
+            raise
+
+        # create some streams connected to the backend
+        cls.ccd = model.getComponent(role="ccd")
+        cls.light = model.getComponent(role="light")
+        cls.light_filter = model.getComponent(role="filter")
+        cls.ebeam = model.getComponent(role="e-beam")
+        cls.sed = model.getComponent(role="se-detector")
+        cls.FM_focuser = TMCLController(
+            name="test_FM_focus",
+            role="focus",
+            port="/dev/fake3",
+            axes=["", "", "z"],
+            ustepsize=[0, 0, 1e-6],
+            rng=[
+                [-3000e-6, 3000e-6],
+            ],
+            refproc="Standard",
+        )
+
+        cls.SEM_focuser = TMCLController(
+            name="test_SEM_focus",
+            role="focus",
+            port="/dev/fake3",
+            axes=["", "", "z"],
+            ustepsize=[0, 0, 0.5e-4],
+            rng=[
+                [-0.5, 0.5],
+            ],
+            refproc="Standard",
+        )
```

Comment:
Why defining additional components If you are using the enzel simulator? It already has focuser for both FM & SEM.

Reply:
Indeed good point